TYPO3.Neos/Documentation/CreatingASite/TypoScript/InsideTypoScript.rst

```diff
@@ -392,6 +392,53 @@ To sum it up: When implementing a TypoScript object, it should not access its co
 directly, but instead use a property. In the TypoScript object's prototype, a default mapping
 between a context variable and the prototype can be set up.
 
+Default Context Variables
+=========================
+
+Neos exposes some default variables to the TypoScript context that can be used to control page rendering
+in a more granular way.
+
+* ``node`` can be used to get access to the current node in the node tree and read its properties, such as::
```

Comment:
there should be a description of how `node` gets overridden down the rendering nesting